app/Services/Contact/Contact/CreateContact.php

```diff
@@ -161,4 +170,30 @@ private function updateDeceasedInformation(array $data, Contact $contact)
             'add_reminder' => $this->nullOrValue($data, 'deceased_date_add_reminder'),
         ]);
     }
+
+    /**
+     * Add an audit log.
+     *
+     * @param array $data
+     * @param Contact $contact
+     * @return void
+     */
+    private function log(array $data, Contact $contact): void
+    {
+        $author = User::find($data['author_id']);
+
+        LogAccountAudit::dispatch([
+            'action' => 'contact_created',
+            'account_id' => $author->account_id,
+            'about_contact_id' => $contact->id,
+            'author_id' => $author->id,
+            'author_name' => $author->name,
+            'audited_at' => Carbon::now(),
```

Comment:
```suggestion
            'audited_at' => now(),
```
use `now()` helper, like everywhere else on our application

Reply:
Good catch!